### PR TITLE
Extension docs

### LINKED ISSUE #1253: Title bar of https://duckdb.org/docs/extensions/autocomplete is off
As per title, https://duckdb.org/docs/extensions/autocomplete looks off, unsure but looks something CSV related

### PR DESCRIPTION
**Pull request by szarnyasg**
Fixes #1253

### CODE REVIEW

File: docs/extensions/httpfs.md [hunk 1]
```diff
@@ -1,29 +1,28 @@
...
 The `httpfs` extension is an autoloadable extension implementing a file system that allows reading remote/writing remote files. For plain HTTP(S), only file reading is supported. For object storage using the S3 API, the `httpfs` extension supports reading/writing/globbing files.
 
 The `httpfs` extension will be, by default, autoloaded on first use of any functionality exposed by this extension.
 If you prefer to explicitly install and load this extension, you can always run `INSTALL httpfs` on first use and issue `LOAD httpfs` at the start of every session.
 
-## HTTP(S)
+## Running Queries over HTTP(S)
 
-With the `httpfs` extension, it is possible to directly query files over the HTTP(S) protocol. This currently works for CSV, JSON, and Parquet files.
+With the `httpfs` extension, it is possible to directly query files over the HTTP(S) protocol. This currently works for CSV, JSON, and Parquet files, and provides read-only access.
```
Comment (carlopi) [inline docs/extensions/httpfs.md:13]:
```suggestion
With the `httpfs` extension, it is possible to directly query files over the HTTP(S) protocol. This works for all files supported by DuckDB or its various extensions, and provides read-only access.
```

Currently the only non-working format are sqlite databases via sqlite_scanner (but PR is up to fix it in sqlite_scanner repo)

File: docs/extensions/httpfs.md [hunk 2]
```diff
@@ -1,29 +1,28 @@
...
 
 ```sql
 SELECT * FROM 'https://domain.tld/file.extension';
```
Comment (carlopi) [inline docs/extensions/httpfs.md:16]:
```suggestion
SELECT * FROM 'https://domain.tld/file.extension';
```
Should this be a real working example?

File: docs/data/parquet/overview.md [hunk 3]
```diff
@@ -3,6 +3,7 @@ layout: docu
 title: Parquet Loading
```
Comment (carlopi) [inline docs/data/parquet/overview.md:3]:
```suggestion
title: Reading Parquet files
```
I think "Loading" should be reserved to extensions, where it's a keyword, but that's for another day.

**Review by carlopi — APPROVED** [review]
Potentially the httpfs can also link to AWS extension, that gives automatic access to credentials.

File: docs/extensions/httpfs.md (hunk 2 above)
Comment (carlopi) [reply docs/extensions/httpfs.md:16]: Also for another day likely.

Comment (szarnyasg) [reply docs/extensions/httpfs.md:16]: Eventually, yes :)

**Pull request merged**